README.md

```diff
@@ -4,6 +4,9 @@ A Terraform Module to set up a pre-configured EC2 SSH Bastion Host.
 
 For the manual setup, see the [Materialize SSH Tunnel](https://materialize.com/docs/ops/network-security/ssh-tunnel) documentation.
 
+> **Warning**
+> This is provided on a best-effort basis and Materialize cannot offer support for this module
```

Comment:
How about: 
Attention: 
Materialize has provided this provider as a starting point to provide baseline security infrastructure using the latest Ubuntu 20 image. It is the user's responsibility to ensure the ec2 bastion host is current with security patches once in service and to take other appropriate steps such as properly securing bastion ssh keys.

Reply:
Love the Warning, keep it!